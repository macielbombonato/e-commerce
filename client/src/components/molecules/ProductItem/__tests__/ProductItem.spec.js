import createTestContext from 'react-cosmos-test/enzyme';

import fixture from '../ProductItem.fixture';

const { mount, getWrapper } = createTestContext({ fixture });

beforeEach(mount);

describe('#ProductItem', () =>
  it('renders component', () => {
    expect(getWrapper()).toMatchSnapshot();
  }));
