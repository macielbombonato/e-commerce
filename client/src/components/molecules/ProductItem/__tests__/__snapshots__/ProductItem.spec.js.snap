// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#ProductItem renders component 1`] = `
<ProductItem
  product={
    Object {
      "description": "description",
      "id": "123",
      "image": "",
      "name": "product",
      "price": 100,
    }
  }
>
  <div
    className="mt-3"
  >
    <Link
      replace={false}
      to="/product/123"
    >
      <a
        href="/product/123"
        onClick={[Function]}
      >
        <Card
          tag="div"
        >
          <div
            className="card"
          >
            <CardImg
              src="http://localhost:1337/images/"
              tag="img"
              width="100%"
            >
              <img
                className="card-img"
                src="http://localhost:1337/images/"
                width="100%"
              />
            </CardImg>
            <CardBody
              tag="div"
            >
              <div
                className="card-body"
              >
                <CardTitle
                  tag="h4"
                >
                  <h4
                    className="card-title"
                  >
                    product
                  </h4>
                </CardTitle>
                <CardText
                  tag="p"
                >
                  <p
                    className="card-text"
                  >
                    description
                  </p>
                </CardText>
                <CardSubtitle
                  tag="h6"
                >
                  <h6
                    className="card-subtitle"
                  >
                    R$ 
                    100.00
                  </h6>
                </CardSubtitle>
              </div>
            </CardBody>
          </div>
        </Card>
      </a>
    </Link>
  </div>
</ProductItem>
`;
